Mage.Verify/src/test/java/mage/verify/VerifyCardDataTest.java

```diff
@@ -2086,6 +2085,69 @@ private void checkSupertypes(Card card, MtgJsonCard ref) {
         }
     }
 
+    // There are many cards that use the word "target" or "targets" in reference to spells/abilities that target rather than actually themselves having a target.
+    // Examples include Wall of Shadows, Psychic Battle, Coalition Honor Guard, Aboleth Spawn, Akroan Crusader, Grip of Chaos, and many more
+    Pattern singularTargetRegexPattern = Pattern.compile("\\b(?<!(can|the|that|could|each|single|its) )target\\b");
+    Pattern pluralTargetsRegexPattern = Pattern.compile("\\b(?<!(new|the|that|copy|choosing|it|more|spell|or|changing|legal) )targets\\b");
+
+    // Note that the check includes reminder text, so any keyword ability with reminder text always included in the card text doesn't need to be added above
```

Comment:
> Note that the check includes reminder text 

Green words or big cards don't have reminder texts, but still targeted. Is it processing in your code?

Reply:
I'm not sure I understand your question correctly. Cards that have ever been printed with reminder text always have it in their oracle text, whether or not the printed card has it.

Reply:
Try to some research on scryfall: `fo:target -o:target`

Looks like there are many use cases for it -- all that ability words must be used in analyse:
![shot_250622_091304](https://github.com/user-attachments/assets/b4634a5e-1325-4805-9c66-98f54ae7d13e)

Reply:
The point of that doc instruction is that abilities where every card printed has the reminder text printed as well don't need to be added. Examples include Provoke, Support, and Soulshift. The inclusion of reminder text in the oracle text is a per-card thing. Backup, for example, is only in the list because of Doomskar Warrior.

(The original doc comment was wrong, it incorrectly had "above" but I already fixed that as can be seen in your screenshot)